OPCB/initPlayerLocal.sqf

```diff
@@ -33,8 +31,11 @@ tank_spawner addAction ["<t color='#FF0000'>I want my vehicle removed!</t>","[]
 
 _uid = getPlayerUID player;
 _SOAR = ["76561198117073327","76561198142692277","76561198067590754","76561198059583284","76561199005382007"];//76561198142692277 -Alex. K., 76561198117073327 - A.Randy,   76561198059583284 - Vittex, 76561198067590754 - Mas Pater, 76561199005382007 - W.Frost
-if (_uid in _soar) then {player setVariable ["SOAR",1]};
-player addEventHandler ["GetInMan",{[_this select 0,_this select 1, _this select 2] call BADCO_role_check;}];
+if (_uid in _soar) then {
+	player setVariable ["SOAR",1];
+};
+
+player addEventHandler ["GetInMan",{[_this select 0,_this select 1, _this select 2] call BADCO_fnc_classCheck;}];
 if (typeof player == "rhsusf_usmc_marpat_d_uav" || typeOf player == "rhsusf_airforce_jetpilot") then 
```

Comment:
I don't know if we would ever change this even on other maps since everyone is going to change their outfit anyway. But it would be wise to parametrise this for future-proofing...

Reply:
Again, I'm talking about player unit classnames. But it seems this is a problem in multiple files, so we can open an issue specifically for this. It's not at all urgent.